managed/src/main/resources/reference.conf

```diff
@@ -88,6 +88,8 @@ yb {
     default_smtp_port_ssl = 465
     debug_email = false
   }
+
+  max_tasks = 2000
```

Comment:
You also need to whitelist it in:
```
runtime_config {
  included_paths = [ 
   // here
  ] 
  ...
}

 ```

Thats where runtime config controller looks to determine if a config key can be used as a runtime configuration.